plugin/pkg/admission/security/podsecuritypolicy/admission_test.go

```diff
@@ -349,43 +349,35 @@ func TestAdmitPreferNonmutating(t *testing.T) {
 		pod                   *kapi.Pod
 		podBeforeUpdate       *kapi.Pod
 		psps                  []*extensions.PodSecurityPolicy
-		shouldPassAdmit       bool
```

Comment:
I'm actually surprised we don't have any tests where this is false... isn't that a test gap we should fix?

Reply:
As far I understand `TestAdmitPreferNonmutating` is focused only on mutating/non-mutating behavior. The cases when a pod is being rejected probably (*) are tested in other tests.

Reply:
@liggitt Could we proceed with merging this PR?